src/main/docker/app.yml

```diff
@@ -0,0 +1,14 @@
+version: '2'
```

Comment:
As this is the registry itself, this file must be changed as it doesn't need to register itself to another registy.